content/docs/v2/_dev/operations/monitoring.md

```diff
@@ -150,20 +150,416 @@ Implement health checks for:
 - Monitor storage backend health according to your storage system's documentation
 - Check network connectivity between components
 
+
+### Internal Observability in Jaeger v2
+
+Jaeger v2 inherits comprehensive observability capabilities from the OpenTelemetry Collector. You can monitor Jaeger's internal operations through:
```

Comment:
This introduction doesn't make sense here. The whole page is about monitoring, and it already has a certain structure. The intro is not bad, it's just in the wrong place, should be at the top of the page. In other words, you need to treat the whole page as holistically a single document, with logical and non-repeating structure. The new content may need to be interspersed throughout the existing sections. You can ask AI to help with organization.